src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -26,7 +26,7 @@ export function SearchFilter() {
     </div>
   ));
 
-  return (
+  return searchFilters.length > 0 ? (
```

Comment:
prevent the search filter DOM elements from rendering if there are no search filters